aidial_adapter_openai/legacy_completions.py

```diff
@@ -0,0 +1,102 @@
+from aidial_sdk.exceptions import HTTPException as DialException
+from fastapi import Response
+from fastapi.responses import StreamingResponse
+from openai import Completion
+from openai.openai_object import OpenAIObject
+
+from aidial_adapter_openai.constant import DEFAULT_TIMEOUT
+from aidial_adapter_openai.env import COMPLETION_DEPLOYMENTS_PROMPT_TEMPLATES
+from aidial_adapter_openai.utils.auth import get_auth_header
+from aidial_adapter_openai.utils.exceptions import handle_exceptions
+from aidial_adapter_openai.utils.sse_stream import to_openai_sse_stream
+from aidial_adapter_openai.utils.streaming import map_stream
+
+
+def convert_completion_to_chat_completion(item: OpenAIObject, role="assistant"):
+    item_dict = item.to_dict_recursive()
+    return {
+        "object": "chat.completion",
+        "id": item_dict["id"],
+        "model": item_dict["model"],
+        "created": item_dict["created"],
+        "usage": item_dict["usage"],
+        "choices": [
+            {
+                "index": choice["index"],
+                "finish_reason": choice["finish_reason"],
+                "message": {
```

Comment:
In the streaming mode the `message` field is called `delta`.

Reply:
fixed